src/Nox.Lib/Presentation/Api/Swagger/RelatedEndpointsDocumentFilter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.OpenApi.Models;
+using Nox.Solution;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Nox.Presentation.Api.Swagger;
+
+public class RelatedEndpointsDocumentFilter : IDocumentFilter
```

Comment:
for having a consistent name in middleware. Internal and sealed..
```suggestion
internal sealed class RelatedEntityRoutingDocumentFilter : IDocumentFilter
```